tuxtape-server/src/database/connection.rs

```diff
@@ -0,0 +1,33 @@
+use diesel::prelude::*;
+use diesel::{Connection, MultiConnection, PgConnection, SqliteConnection};
+
+/// The allowed database backends
+#[derive(Clone, Debug, clap::ValueEnum)]
+pub enum DatabaseBackend {
+    Pg,
+    Sqlite,
+}
+
+/// A wrapper to support multiple database backends.
+#[derive(MultiConnection)]
+pub enum AnyConnection {
```

Comment:
Can we just call this `Connection`?
```suggestion
pub enum Connection {
```

Reply:
Unfortunately, I don't think this can be done elegantly. `Connection` is a `diesel` type that gets included in `diesel::prelude::*`, and we want to use that glob import in a few places. I took the `AnyConnection` name from [their docs](https://docs.rs/diesel/latest/diesel/derive.MultiConnection.html), so this seems idiomatic for the library.

Reply:
imo we should not be using wildcards for imports, but that's another point. `AnyConnection` is ok.

Reply:
I agree in principle, but I've seen a few libraries who specifically recommend glob imports of `::prelude`. (`ratatui` is another one). My understanding is it's idiomatic.